Specs/Scene/PostProcessStageLibrarySpec.js

```diff
@@ -304,10 +304,12 @@ describe(
         origin,
         new HeadingPitchRoll()
       );
+      scene.postProcessStages.ambientOcclusion.enabled = false;
```

Comment:
Was this intentional?

Reply:
Not intentional, I think it might have been an accidental middle-click (paste) in Linux :sweat_smile: